src/EditorFeatures/CSharpTest/AddAccessibilityModifiers/AddAccessibilityModifiersTests.cs

```diff
@@ -143,13 +190,53 @@ internal enum E
+                        // Test0.cs(32,18): error CS0540: 'C.I.M3()': containing type does not implement interface 'I'
+                        DiagnosticResult.CompilerError("CS0540").WithSpan(32, 18, 32, 19).WithArguments("Outer.Inner1.Inner2.C.Outer.Inner1.Inner2.I.M3()", "Outer.Inner1.Inner2.I"),
+                        // Test0.cs(33,26): error CS0751: A partial method must be declared within a partial class, partial struct, or partial interface
+                        DiagnosticResult.CompilerError("CS0751").WithSpan(33, 26, 33, 28),
+                        // Test0.cs(33,26): error CS0759: No defining declaration found for implementing declaration of partial method 'C.M4()'
+                        DiagnosticResult.CompilerError("CS0759").WithSpan(33, 26, 33, 28).WithArguments("Outer.Inner1.Inner2.C.M4()"),
+                        // Test0.cs(37,17): error CS0540: 'C.I.P3': containing type does not implement interface 'I'
+                        DiagnosticResult.CompilerError("CS0540").WithSpan(37, 17, 37, 18).WithArguments("Outer.Inner1.Inner2.C.Outer.Inner1.Inner2.I.P3", "Outer.Inner1.Inner2.I"),
+                        // Test0.cs(39,39): error CS0501: 'C.this[int].get' must declare a body because it is not marked abstract, extern, or partial
+                        DiagnosticResult.CompilerError("CS0501").WithSpan(39, 39, 39, 42).WithArguments("Outer.Inner1.Inner2.C.this[int].get"),
+                        // Test0.cs(40,41): error CS0501: 'C.this[string].get' must declare a body because it is not marked abstract, extern, or partial
+                        DiagnosticResult.CompilerError("CS0501").WithSpan(40, 41, 40, 44).WithArguments("Outer.Inner1.Inner2.C.this[string].get"),
+                        // Test0.cs(41,17): error CS0540: 'C.I.this[bool]': containing type does not implement interface 'I'
+                        DiagnosticResult.CompilerError("CS0540").WithSpan(41, 17, 41, 18).WithArguments("Outer.Inner1.Inner2.C.Outer.Inner1.Inner2.I.this[bool]", "Outer.Inner1.Inner2.I"),
+                        // Test0.cs(41,34): error CS0501: 'C.I.this[bool].get' must declare a body because it is not marked abstract, extern, or partial
+                        DiagnosticResult.CompilerError("CS0501").WithSpan(41, 34, 41, 37).WithArguments("Outer.Inner1.Inner2.C.Outer.Inner1.Inner2.I.this[bool].get"),
+                        // Test0.cs(46,19): error CS0246: The type or namespace name 'Action' could not be found (are you missing a using directive or an assembly reference?)
+                        DiagnosticResult.CompilerError("CS0246").WithSpan(46, 19, 46, 25).WithArguments("Action"),
+                    },
+                },
+            }.RunAsync();
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddAccessibilityModifiers)]
         public async Task TestRefStructs()
         {
-            await TestInRegularAndScriptAsync(@"
```

Comment:
Testing in script code is not currently supported, because diagnostics in script code fail to include the document path in the diagnostic.

Reply:
this is fine with me.